src/main/java/lt/vu/application/user/service/UpdateUserInfoService.java

```diff
@@ -0,0 +1,28 @@
+package lt.vu.application.user.service;
+
+import lt.vu.application.address.factory.AddressFactory;
+import lt.vu.persistence.orm.entities.User;
+import lt.vu.persistence.orm.repository.UserRepository;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class UpdateUserInfoService {
```

Comment:
Not sure about the naming; Put or Update?

Reply:
Put is a low-level (http) implemention detail. A service class is a high level implementation. So in this case, `update` is the better word. Also, I think it's more clear when services start with model/domain name, not a verb - `UserInfoUpdateService` - a service that updates UserInfo.